lib/bundle/cask_dumper.rb

```diff
@@ -16,25 +17,33 @@ def casks
             .uniq
     end
 
-    def dump(casks_required_by_formulae)
+    def dump(casks_required_by_formulae, describe: false)
       [
-        (casks & casks_required_by_formulae).map { |cask| "cask \"#{cask}\"" }.join("\n"),
-        (casks - casks_required_by_formulae).map { |cask| "cask \"#{cask}\"" }.join("\n"),
+        (casks & casks_required_by_formulae).map { |cask| cask_line(cask, describe) }.join("\n"),
+        (casks - casks_required_by_formulae).map { |cask| cask_line(cask, describe) }.join("\n"),
       ]
     end
 
-    def formula_dependencies(cask_list)
-      return [] unless cask_list.present?
+    def cask_line(cask, describe)
+      desc = cask_info&.flat_map.find { |info| info["token"] == cask }&["desc"]
+      comment = "# #{desc}\n" if describe && !desc.blank?
```

Comment:
```suggestion

      comment = if describe
        desc = cask_info.flat_map.find { |info| info["token"] == cask }&["desc"]
        "# #{desc}\n" if desc.present?
      end
```

`cask_info&` is unnecessary because `cask_info` will never return `nil`. If it did: `formula_dependencies` would break too.

Reply:
This was not resolved.